homeassistant/components/xiaomi_miio/manifest.json

```diff
@@ -2,7 +2,10 @@
   "domain": "xiaomi_miio",
   "name": "Xiaomi miio",
   "documentation": "https://www.home-assistant.io/integrations/xiaomi_miio",
-  "requirements": ["construct==2.9.45", "python-miio==0.4.8"],
+  "requirements": [
+    "construct==2.9.45",
+    "python-miio==0.5.0"
```

Comment:
Please bump this to 0.5.0.1, as that bump will be needed soon enough due to a problem with packaging of 0.5.0.

Otherwise this seems to be good to go from my side.

Reply:
Updated